docs/Api/MetricsApi.md

```diff
@@ -113,7 +113,7 @@ $apiInstance = new MuxPhp\Api\MetricsApi(
     $config
 );
 $metric_id = video_startup_time; // string | ID of the Metric
-$timeframe = array('timeframe_example'); // string[] | Timeframe window to limit results by. Must be provided as an array query string parameter (e.g. timeframe[]=).  Accepted formats are...    * array of epoch timestamps e.g. `timeframe[]=1498867200&timeframe[]=1498953600`   * duration string e.g. `timeframe[]=24:hours or timeframe[]=7:days`
+$timeframe = array('timeframe' => [1498867200, 1498953600]); // string[] | Timeframe window to limit results by. Must be provided as an array query string parameter (e.g. timeframe[]=).  Accepted formats are...    * array of epoch timestamps e.g. `timeframe[]=1498867200&timeframe[]=1498953600`   * duration string e.g. `timeframe[]=24:hours or timeframe[]=7:days`
```

Comment:
@philcluff Does this change present the same issue with release integration testing?

Reply:
Hey @timmch, this is fine, but this change would get lost when we regenerate the SDK from our OpenAPI specifications, and there's not a whole lot we can do to address that unfortunately immediately. We are working on a larger overhaul of all our SDKs at the moment, and I'll make sure to bring this up specifically as an example with them.

Reply:
Gotcha, thanks for the info @philcluff!